### PR TITLE
feat: separate project set up pages

### PR DESCRIPTION
**Pull request by nathaliabj**
Closes: #2288 

### Description:

This pr creates a separate card to choose warehouse and other "page" to set up project.

Screenshot 2022-06-07 at 12 18 14
Screenshot 2022-06-07 at 12 18 07
Screenshot 2022-06-07 at 12 17 51

### CODE REVIEW

**Comment by rephus** [comment]
Is it possible to load the icons in a way so they don't flash and get reloaded every time ? 

file:///home/rephus/Videos/Peek%202022-06-08%2008-21.gif


image

File: packages/frontend/src/pages/CreateProject.tsx [hunk 1]
```diff
@@ -1,44 +1,123 @@
...
+    Subtitle,
+    Title,
+    WarehouseButton,
+    WarehouseGrid,
+    WarehouseIcon,
+    Wrapper,
+} from './CreateProject.styles';
+
+const WarehouseTypeLabels = [
+    { label: 'Bigquery', key: WarehouseTypes.BIGQUERY, icon: './bigquery.png' },
```
Comment (rephus) [inline packages/frontend/src/pages/CreateProject.tsx:23]: small label typo 

```suggestion
    { label: 'BigQuery', key: WarehouseTypes.BIGQUERY, icon: './bigquery.png' },
```

**Review by rephus — CHANGES_REQUESTED** [review]
For some reason, we are missing the `warehouseSchemaInput` when selecting the warehouse. for example, on Databricks, you should be able to see an additional `schema` input in the `dbt` form. 

See this PR / gif for more details https://github.com/lightdash/lightdash/pull/2325

**Comment by nathaliabj** [comment]
Closing as opening #2354 where i have fixed the above comments and cherry-picked the commits to fix merge issues on this branch

**Pull request closed without merging**